### ISSUE TITLE
Generalize documentation instructions for SSH deploy keys

### ISSUE THREAD

**Issue by charleskawczynski** [JuliaDocs/Documenter.jl#2580]
The docs read (found by @Sbozzolo):

> It is also possible to authenticate using a SSH deploy key, just as described in the [SSH Deploy Keys section for Travis CI](https://documenter.juliadocs.org/stable/man/hosting/#travis-ssh).

I believe that the instructions are essentially the same for GitHub Actions, but this should be made clear in the docs

**Comment by goerz** [comment]
Yeah, the docs need a rewrite. It's on the TODO list, but nobody has had the time to do it

**Comment by fingolfin** [comment]
Resolved by PR #2968
